picard/ui/mainwindow.py

```diff
@@ -1080,6 +1096,14 @@ def update_actions(self):
         self.tags_from_filenames_action.setEnabled(bool(files))
         self.track_search_action.setEnabled(have_objects)
 
+        try:
+            if (single.linked_files[0].filename in match_details_log.files):
+                self.track_match_details_action.setEnabled(have_objects)
+            else:
+                self.track_match_details_action.setEnabled(False)
+        except Exception:
+            self.track_match_details_action.setEnabled(False)
```

Comment:
What about storing boolean in a variable and call `self.track_match_details_action.setEnabled(is_enabled)`?